src/containers/auth/resetPassword/index.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { PureComponent } from 'react';
+import styles from './styles.module.scss';
+import { resetApprove, resetPassword } from '../../../services/auth';
+import Spinner from '../../../components/spinner/index';
+import validateAuth from '../../../validation/auth';
+
+class ResetPassword extends PureComponent {
+    state = {
+        success: false,
+        status: null,
+        password: null,
+        passwordConfirm: null,
+        linkId: null,
+        message: null
+    }
+
+    componentDidMount() {
+        const params = new URLSearchParams(location.search);
+        this.sendApprove(params.get('id'));
+    }
+
+    sendApprove = async (linkId) => {
+        const { status, data } = await resetApprove(linkId);
+        if (data.name === 'Error') {
+            this.setState({
+                status: 400,
+                linkId: null,
+            });
+        } else {
+            this.setState({
+                status,
+                linkId
+            });
+        }
+    }
+
+    sendReset = async (e) => {
```

Comment:
Remove async

Reply:
Without async, it doesn't work. Because of Axios return a promise and it needs to wait to resolve one.